requirements.txt

```diff
@@ -0,0 +1,12 @@
+# Note that the cmake build will consume this for you if you have virtualenv
+# installed. If not you'll need to manually setup the correct python3 environment
+# and install these requirements.
+
+# libuavcan has a single python dependency although pydsdlgen itself will
+# have one or more dependencies including (but not necessarily limited to)
+# pydsdl
+
+pydsdlgen >= 1.0
```

Comment:
Did you mean `pydsdlgen < 2.0`?

Reply:
Not exactly. pydsdlgen < 1.0 won't work with this.  I'll change this to:

`pydsdlgen >= 1.0, < 2.0`